docs/_integration-agent/file-folder-listener/local-folder-listener.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Local Folder Listener
 nav_order: 1
-parent: File Folder Listener
```

Comment:
I think you want to leave this file alone.  The local file listener docs under File Folder Listener should remain unchanged.